docs/index.html

```diff
@@ -105,6 +105,9 @@ <h1 id="iban-info-title" class="text-2xl font-bold text-primary mb-4">IBAN Infor
             </div>
         </main>
             <footer class="w-full max-w-2xl mx-auto text-center mt-6 pt-8 border-t border-gray-200 text-gray-600">
+                <p id="disclaimer-text" class="text-sm mb-4 text-gray-700 bg-yellow-50 border border-yellow-200 rounded-md p-3">
```

Comment:
Add it as a "footer" of the content itself (`<main>`), to keep the footer related to the website itself